data_store/db_helper.py

```diff
@@ -0,0 +1,9 @@
+statements = {
```

Comment:
Is this useful somewhere in this feature or will be used afterward?

Reply:
This is for web version part, will be used later